recipes/netpbm/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "10.68" %}
+
+package:
+  name: netpbm
+  version: {{ version }}
+
+source:
+  # See http://netpbm.svn.sourceforge.net/viewvc/netpbm/
+  svn_url: http://svn.code.sf.net/p/netpbm/code/advanced
+  svn_rev: 2294
+
+build:
+    number: 0
+    skip: true      # [win]
+
+requirements:
+  build:
+    - jpeg
+    - libtiff
+    - libpng
+    - zlib
+    - lxml
+    - svn
+  run:
+    - jpeg
+    - libtiff
+    - libpng
+    - zlib
+    - lxml
+test:
+  commands:
+    # There are a ton of commands, but just check one to make sure it works.
+    - pnmtopng --version
+
+about:
+  home: http://netpbm.sourceforge.net/
+  license: GPL
```

Comment:
Wikipedia suggest they all fall under DFSG. Do you have any thoughts on this, @ocefpaf? This one is pretty hairy just because of how much software is here and how many licenses are used.